app/graphql/types/integration_collection_mappings/netsuite/update_input.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Types
+  module IntegrationCollectionMappings
+    module Netsuite
+      class UpdateInput < Types::BaseInputObject
+        graphql_name 'UpdateNetsuiteIntegrationCollectionMappingInput'
+
+        argument :integration_id, ID, required: false
+        argument :netsuite_account_code, String, required: false
+        argument :netsuite_id, String, required: false
+        argument :netsuite_name, String, required: false
+      end
```

Comment:
I think we miss ID here so that we can find the record on BE side. Did you have something else in mind?